packages/cli/src/tools/packageManager.js

```diff
@@ -61,8 +67,34 @@ export function uninstall(packageNames: Array<string>, options?: Options) {
     : executeCommand('npm', ['uninstall', ...packageNames, '--save'], options);
 }
 
-export function installAll(options?: Options) {
-  return shouldUseYarn(options)
-    ? executeCommand('yarn', ['install'], options)
-    : executeCommand('npm', ['install'], options);
+export function installCocoaPods() {
+  return executeCommand('sudo', ['gem', 'install', 'cocoapods'], {
+    silent: false,
+  });
+}
+
+export async function installPods(options?: Options) {
+  try {
+    await commandExists('pod');
+  } catch (err) {
+    return {error: ERRORS.noCocoaPods};
+  }
+
+  process.chdir('ios');
+
+  try {
+    return await executeCommand('pod', ['install'], options);
+  } catch (err) {
+    return {error: ERRORS.failedToRunPodInstall};
```

Comment:
There's a special CLIError to which you can pass a `stderr` from the `executeCommand` (or the `err.message` in this case). That way, we are not going to swallow an original error message and could see it when running CLI with `--verbose` command.

In case you need a special error here, you would need to subclass `CLIError` to give it a different name

Reply:
However, you might not need to subclass errors if you decide to not do it inside PackageManager :)